pkg/apis/kops/componentconfig.go

```diff
@@ -384,6 +384,9 @@ type KubeAPIServerConfig struct {
 	KubeletPreferredAddressTypes []string `json:"kubeletPreferredAddressTypes,omitempty" flag:"kubelet-preferred-address-types"`
```

Comment:
@justinsb explained that we can have API values on components that don't have legacy yaml stuff, and it will add the flags automatically.